### DISCUSSION TITLE
Problems DNS

### DISCUSSION THREAD

**Discussion by maiconluizanschau** [vercel/community#1962] [Help]
### Summary

I can't find documentation on how to update a domain that was acquired by google, but I can't find where I can make this change, I contacted google and it said that it would have to be in your solution.

**Comment by amyegan** [comment]
Hi @maiconluizanschau! You can find more information about [adding & configuring a Custom Domain](https://vercel.com/docs/concepts/projects/domains/add-a-domain#) in our documentation. 

Google has it's own documentation about managing [A records](https://support.google.com/a/answer/2576578#zippy=%2Cconfigure-a-records-now), [CNAME records](https://support.google.com/a/answer/112037#zippy=%2Cset-up-cname-records-now), and [name servers](https://support.google.com/domains/answer/3290309?hl=en)
